Clean up pre auth documentation (#469)
centrapay/centrapay-docs: src/api/payment-requests/payment-requests.md

| expiry            | [Payment Request][] wasn't paid before time out.                  |
| accept-condition  | A [Payment Condition][] was accepted.                             |
| decline-condition | A [Payment Condition][] was declined.                             |
| release           | Remaining funds on a Pre Auth were returned to the shopper.       |

> | confirmation      | Funds on a Pre Auth have been drawn down on.                      |

Reviewer: ```suggestion
| confirmation      | Unspent funds on a Pre Auth have been returned.                      |
```

> | confirmation      | Funds on a Pre Auth have been drawn down on.                      |

Reviewer: Can we move confirmation above release?

| expiry            | [Payment Request][] wasn't paid before time out.                  |
| accept-condition  | A [Payment Condition][] was accepted.                             |
| decline-condition | A [Payment Condition][] was declined.                             |

> | confirmation      | Funds on a Pre Auth have been drawn down on.                      |

Reviewer 2: ```suggestion
| confirmation      | Draw down against funds from Pre Auth Authorization.                      |
```

Author: This suggestion is not consistent with the tense of the descriptions above. For this reason I am not going to accept this change.

> | release           | Remaining funds on a Pre Auth were returned to the shopper.       |

Reviewer 2: ```suggestion
| release           | Pre Auth has been finalised and any remaing funds from Authorization have been returned.       |
```
